src/classes/Api/Handlers/RepeaterHandler.php

```diff
@@ -0,0 +1,151 @@
+<?php
+/**
+ * Repeater Handler class
+ *
+ * @package notification
+ */
+
+namespace BracketSpace\Notification\Api\Handlers;
+
+/**
+ * RepeaterHandler class
+ *
+ * @action
+ */
+class RepeaterHandler {
+
+	/**
+	 * Forms field data
+	 *
+	 * @since [Next]
+	 * @param array $data Field data.
+	 * @return array
+	 */
+	public function form_field_data( $data ) {
+
+		$fields = [];
+
+		foreach ( $data as $field ) {
+
+			$sub_field = [];
+
+			$sub_field['options']        = $field->options;
+			$sub_field['pretty']         = $field->pretty;
+			$sub_field['label']          = $field->label;
+			$sub_field['checkbox_label'] = $field->checkbox_label;
+			$sub_field['name']           = $field->name;
+			$sub_field['description']    = $field->description;
+			$sub_field['section']        = $field->section;
+			$sub_field['disabled']       = $field->disabled;
+			$sub_field['css_class']      = $field->css_class;
+			$sub_field['id']             = $field->id;
+			$sub_field['placeholder']    = $field->placeholder;
+			$sub_field['nested']         = $field->nested;
+			$sub_field['type']           = strtolower( str_replace( 'Field', '', $field->field_type_html ) );
+
+			if ( $field->fields ) {
+				$sub_field['fields'] = $this->form_field_data( $field->fields );
+			}
+
+			array_push( $fields, $sub_field );
+
+		}
+
+		return $fields;
+
+	}
+
+	/**
+	 * Gets field values
+	 *
+	 * @since [Next]
+	 * @param int    $post_id Post id.
+	 * @param string $carrier Carrier slug.
+	 * @param string $field Field slug.
+	 * @return array
+	 */
+	public function get_values( $post_id, $carrier, $field ) {
+		$notification = notification_adapt_from( 'WordPress', $post_id );
+
+		$carrier = $notification->get_carrier( $carrier );
+
+		if ( $carrier ) {
+			return $carrier->get_field_value( $field );
+		}
+
+		return [];
+	}
+
+	/**
+	 * Checks if field is instance of repeater field
+	 *
+	 * @since [Next]
+	 * @param \BracketSpace\Notification\Abstracts\Field $field Form field type.
+	 * @return boolean
+	 */
+	public function check_repeater( \BracketSpace\Notification\Abstracts\Field $field ) {
```

Comment:
Move the fully qualified namespaces to `use` statements. To not duplicate them fix the biggest common name, ie. `BracketSpace\Notification\Defaults\Field` then in the code just use `Field\RecipientsField`